Dockerfile

```diff
@@ -12,24 +12,26 @@
 # the License.
 #
 
-FROM adoptopenjdk/openjdk11:alpine
+FROM eclipse-temurin:11-jdk-alpine as builder
+WORKDIR /build
+RUN apk add --no-cache git
+COPY ./ ./
+RUN ./mvnw install -DskipTests
 
-EXPOSE 8080
-EXPOSE 8081
-EXPOSE 8443
 
-RUN addgroup -S thirdeye && \
-  adduser -S thirdeye -G thirdeye && \
-  chown thirdeye:thirdeye /home/thirdeye
+FROM eclipse-temurin:11-jre-alpine
+RUN addgroup -g 1000 thirdeye && \
+  adduser -u 1000 thirdeye -G thirdeye -D -H
 
 USER thirdeye
-WORKDIR /home/thirdeye
+WORKDIR /app
```

Comment:
appart from being a best practice, what's the reason for changing `/home/thirdeye` to `/app`?

Reply:
Only beauty and conciseness and following common practices.
If you do not like this option, you can return the previous value.

Reply:
yeah we have some [JFR](https://docs.oracle.com/javacomponents/jmc-5-4/jfr-runtime-guide/about.htm#JFRUH170) collection in prod that as of today is written to `/home/thirdeye`.

Some systems consuming the profiling data may assume it's in `/home/thirdeye`. 
Overall that's more risk than benefits, so yes please keep the previous value.